style.css

```diff
@@ -601,11 +600,13 @@ input:focus:invalid {
  #container1 {
     height: 100vh;
     background-color: #1c1a19;
-    background-image: url("images/Illustration-header.png");
+    background-image: url(images/Illustration-header.png);
     background-repeat: no-repeat;
     background-position: right;
+    background-position-y: 0.2rem;
```

Comment:
- [ ] Nice work so far 👍🏿 on your project. As requested by the previous code reviewer this issue has not been resolved yet. Please I kindly suggest you centralize this card just as the Figma design 
- [ ] Your design
![image](https://github.com/Tshepo45/Portfolio-desktop-version/assets/68823331/c9368686-013c-4c36-90ec-639b54feff43)

- [ ] Figma design
![image](https://github.com/Tshepo45/Portfolio-desktop-version/assets/68823331/183ac0af-02ae-4c22-b9b1-4e7afff7c0f4)